src/components/Board/Board.js

```diff
@@ -1,27 +1,45 @@
 import React, { useEffect, useState } from 'react';
 import Grid from '@material-ui/core/Grid';
 import Column from '../Column/Column';
-import * as cardsApi from '../../api/cardsApi';
+import { getBoards } from '../../api/boardsApi';
+import { addNewCard } from '../../services/BoardService';
 
 const Board = () => {
-  const [cards, setCards] = useState([]);
-
+  const [board, setBoard] = useState({ columns: [] });
+  const [error, setError] = useState(false);
   const addCard = (newCard) => {
-    setCards([...cards, newCard]);
+    const newState = addNewCard(board, newCard);
+
+    setBoard({ ...board, newState });
   };
 
   useEffect(() => {
-    cardsApi.getCards().then((cardsResponse) => {
-      setCards(cardsResponse);
+    getBoards().then((boardResponse) => {
+      setBoard(boardResponse);
+    }).catch((e) => {
+      console.log(e.message);
+      setError(true);
     });
   }, []);
 
-  return (
+  const renderGrid = () => (error ? (
+    <>
+      <p>Sorry something went wrong...</p>
+    </>
+  ) : (
     <Grid container spacing={3}>
-      <Grid item xs={3}>
-        <Column cards={cards} addNewCardToBoard={addCard} />
-      </Grid>
+      {board.columns.map((column) => (
+        <Grid item xs={3} key={column.id}>
+          <Column key={column.id} columnProp={column} addNewCardToBoard={addCard} />
+        </Grid>
+      ))}
     </Grid>
+  ));
+
+  return (
+    <div>
```

Comment:
You can use react fragments `<>` instead of the `div` element, so there is no unnecessary markup added.

Reply:
done